Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -77,6 +84,18 @@ void OnDestroy()
             }
         }
 
+        private void ProcessKeyBindings()
+        {
+            foreach (var kvp in KeywordsAndResponses)
+            {
+                if (Input.GetKeyDown(kvp.KeyCode))
```

Comment:
You should change this to GetKeyUp (so the call happens on release of the keyboard key).